{
 "errors": {
  "h5py.tests.hl": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/__init__.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_attribute_create": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_attribute_create.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_attribute_create.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_dataset_getitem": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_dataset_getitem.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_dataset_getitem.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_dataset_swmr": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_dataset_swmr.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_dataset_swmr.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_datatype": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_datatype.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_datatype.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_deprecation": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_deprecation.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_deprecation.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_file": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_file.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_file.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_filters": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_filters.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_filters.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_threads": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_threads.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_threads.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_vds": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_vds/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_vds/__init__.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_vds.test_highlevel_vds": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_vds/test_highlevel_vds.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_vds/test_highlevel_vds.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_vds.test_lowlevel_vds": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_vds/test_lowlevel_vds.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_vds/test_lowlevel_vds.py'",
    ""
   ]
  },
  "h5py.tests.hl.test_vds.test_virtual_source": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_vds/test_virtual_source.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/hl/test_vds/test_virtual_source.py'",
    ""
   ]
  },
  "h5py.tests.old": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/__init__.py'",
    ""
   ]
  },
  "h5py.tests.old.test_attrs": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_attrs.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_attrs.py'",
    ""
   ]
  },
  "h5py.tests.old.test_attrs_data": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_attrs_data.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_attrs_data.py'",
    ""
   ]
  },
  "h5py.tests.old.test_base": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_base.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_base.py'",
    ""
   ]
  },
  "h5py.tests.old.test_dataset": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_dataset.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_dataset.py'",
    ""
   ]
  },
  "h5py.tests.old.test_datatype": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_datatype.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_datatype.py'",
    ""
   ]
  },
  "h5py.tests.old.test_dimension_scales": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_dimension_scales.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_dimension_scales.py'",
    ""
   ]
  },
  "h5py.tests.old.test_file": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_file.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_file.py'",
    ""
   ]
  },
  "h5py.tests.old.test_file_image": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_file_image.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_file_image.py'",
    ""
   ]
  },
  "h5py.tests.old.test_group": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_group.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_group.py'",
    ""
   ]
  },
  "h5py.tests.old.test_h5": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5.py'",
    ""
   ]
  },
  "h5py.tests.old.test_h5d_direct_chunk_write": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5d_direct_chunk_write.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5d_direct_chunk_write.py'",
    ""
   ]
  },
  "h5py.tests.old.test_h5f": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5f.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5f.py'",
    ""
   ]
  },
  "h5py.tests.old.test_h5p": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5p.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5p.py'",
    ""
   ]
  },
  "h5py.tests.old.test_h5t": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5t.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_h5t.py'",
    ""
   ]
  },
  "h5py.tests.old.test_objects": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_objects.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_objects.py'",
    ""
   ]
  },
  "h5py.tests.old.test_selections": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_selections.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_selections.py'",
    ""
   ]
  },
  "h5py.tests.old.test_slicing": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_slicing.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 34, in single_file_extraction",
    "    data = jedi.Script(path=file_name, project=jedi.Project(''.join(top_dir))).complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpgfu_4r7m/lib/python3.7/site-packages/h5py/tests/old/test_slicing.py'",
    ""
   ]
  }
 },
 "symbols": [
  "h5py.AttributeManager",
  "h5py.Dataset",
  "h5py.Datatype",
  "h5py.Empty",
  "h5py.ExternalLink",
  "h5py.File",
  "h5py.Group",
  "h5py.HLObject",
  "h5py.HardLink",
  "h5py.Reference",
  "h5py.RegionReference",
  "h5py.SoftLink",
  "h5py.VirtualLayout",
  "h5py.VirtualSource",
  "h5py._ModuleWrapper",
  "h5py.__version__",
  "h5py._errors",
  "h5py._hl",
  "h5py._hl.attrs",
  "h5py._hl.attrs.AttributeManager",
  "h5py._hl.attrs._lapl",
  "h5py._hl.attrs._lcpl",
  "h5py._hl.base",
  "h5py._hl.base.CommonStateObject",
  "h5py._hl.base.CommonStateObject._lapl",
  "h5py._hl.base.CommonStateObject._lcpl",
  "h5py._hl.base.Empty",
  "h5py._hl.base.HLObject",
  "h5py._hl.base.HLObject.__nonzero__",
  "h5py._hl.base.HLObject.attrs",
  "h5py._hl.base.HLObject.file",
  "h5py._hl.base.HLObject.name",
  "h5py._hl.base.HLObject.parent",
  "h5py._hl.base.HLObject.ref",
  "h5py._hl.base.HLObject.regionref",
  "h5py._hl.base.ItemsViewHDF5",
  "h5py._hl.base.KeysViewHDF5",
  "h5py._hl.base.MappingHDF5",
  "h5py._hl.base.MutableMappingHDF5",
  "h5py._hl.base.ValuesViewHDF5",
  "h5py._hl.base._RegionProxy",
  "h5py._hl.base.default_lapl",
  "h5py._hl.base.default_lcpl",
  "h5py._hl.base.dlapl",
  "h5py._hl.base.dlcpl",
  "h5py._hl.base.guess_dtype",
  "h5py._hl.base.is_empty_dataspace",
  "h5py._hl.base.is_hdf5",
  "h5py._hl.compat",
  "h5py._hl.compat.WINDOWS_ENCODING",
  "h5py._hl.compat._fsdecode",
  "h5py._hl.compat._fsencode",
  "h5py._hl.compat.filename_decode",
  "h5py._hl.compat.filename_encode",
  "h5py._hl.compat.fsdecode",
  "h5py._hl.compat.fsencode",
  "h5py._hl.compat.fspath",
  "h5py._hl.dataset",
  "h5py._hl.dataset.AstypeContext",
  "h5py._hl.dataset.CollectiveContext",
  "h5py._hl.dataset.Dataset",
  "h5py._hl.dataset.MPI",
  "h5py._hl.dataset._LEGACY_GZIP_COMPRESSION_VALS",
  "h5py._hl.dataset.__nonzero__",
  "h5py._hl.dataset._lapl",
  "h5py._hl.dataset._lcpl",
  "h5py._hl.dataset.attrs",
  "h5py._hl.dataset.file",
  "h5py._hl.dataset.make_new_dset",
  "h5py._hl.dataset.make_new_virtual_dset",
  "h5py._hl.dataset.name",
  "h5py._hl.dataset.parent",
  "h5py._hl.dataset.readtime_dtype",
  "h5py._hl.dataset.ref",
  "h5py._hl.dataset.regionref",
  "h5py._hl.datatype",
  "h5py._hl.datatype.Datatype",
  "h5py._hl.datatype.__nonzero__",
  "h5py._hl.datatype._lapl",
  "h5py._hl.datatype._lcpl",
  "h5py._hl.datatype.attrs",
  "h5py._hl.datatype.file",
  "h5py._hl.datatype.parent",
  "h5py._hl.datatype.ref",
  "h5py._hl.datatype.regionref",
  "h5py._hl.dims",
  "h5py._hl.dims.DimensionManager",
  "h5py._hl.dims.DimensionProxy",
  "h5py._hl.dims._lapl",
  "h5py._hl.dims._lcpl",
  "h5py._hl.files",
  "h5py._hl.files.File",
  "h5py._hl.files.__nonzero__",
  "h5py._hl.files._drivers",
  "h5py._hl.files._gcpl_crt_order",
  "h5py._hl.files._lapl",
  "h5py._hl.files._lcpl",
  "h5py._hl.files._set_fapl_fileobj",
  "h5py._hl.files._set_fapl_mpio",
  "h5py._hl.files.attrs",
  "h5py._hl.files.file",
  "h5py._hl.files.hdf5_version",
  "h5py._hl.files.libver_dict",
  "h5py._hl.files.libver_dict_r",
  "h5py._hl.files.make_fapl",
  "h5py._hl.files.make_fcpl",
  "h5py._hl.files.make_fid",
  "h5py._hl.files.mpi",
  "h5py._hl.files.name",
  "h5py._hl.files.parent",
  "h5py._hl.files.ref",
  "h5py._hl.files.regionref",
  "h5py._hl.files.register_driver",
  "h5py._hl.files.registered_drivers",
  "h5py._hl.files.swmr_support",
  "h5py._hl.files.unregister_driver",
  "h5py._hl.filters",
  "h5py._hl.filters.CHUNK_BASE",
  "h5py._hl.filters.CHUNK_MAX",
  "h5py._hl.filters.CHUNK_MIN",
  "h5py._hl.filters.DEFAULT_GZIP",
  "h5py._hl.filters.DEFAULT_SZIP",
  "h5py._hl.filters._COMP_FILTERS",
  "h5py._hl.filters._external_entry",
  "h5py._hl.filters._gen_filter_tuples",
  "h5py._hl.filters._normalize_external",
  "h5py._hl.filters.decode",
  "h5py._hl.filters.encode",
  "h5py._hl.filters.fill_dcpl",
  "h5py._hl.filters.get_filters",
  "h5py._hl.filters.guess_chunk",
  "h5py._hl.group",
  "h5py._hl.group.ExternalLink",
  "h5py._hl.group.Group",
  "h5py._hl.group.Group._gcpl_crt_order",
  "h5py._hl.group.HardLink",
  "h5py._hl.group.SoftLink",
  "h5py._hl.selections",
  "h5py._hl.selections.FancySelection",
  "h5py._hl.selections.PointSelection",
  "h5py._hl.selections.Selection",
  "h5py._hl.selections.SimpleSelection",
  "h5py._hl.selections._RegionProxy",
  "h5py._hl.selections._expand_ellipsis",
  "h5py._hl.selections._handle_simple",
  "h5py._hl.selections._translate_int",
  "h5py._hl.selections._translate_slice",
  "h5py._hl.selections.guess_shape",
  "h5py._hl.selections.select",
  "h5py._hl.selections2",
  "h5py._hl.selections2.ScalarReadSelection",
  "h5py._hl.selections2.read_dtypes",
  "h5py._hl.selections2.read_selections_scalar",
  "h5py._hl.selections2.select_read",
  "h5py._hl.vds",
  "h5py._hl.vds.VDSmap",
  "h5py._hl.vds.VirtualLayout",
  "h5py._hl.vds.VirtualSource",
  "h5py._hl.vds.hdf5_version",
  "h5py._hl.vds.vds_support",
  "h5py._register_converters",
  "h5py._register_lzf",
  "h5py.check_dtype",
  "h5py.enable_ipython_completer",
  "h5py.filters",
  "h5py.get_config",
  "h5py.get_enum",
  "h5py.get_vlen",
  "h5py.h5a",
  "h5py.h5d",
  "h5py.h5ds",
  "h5py.h5f",
  "h5py.h5fd",
  "h5py.h5g",
  "h5py.h5p",
  "h5py.h5py_warnings",
  "h5py.h5py_warnings.H5pyDeprecationWarning",
  "h5py.h5py_warnings.H5pyWarning",
  "h5py.h5py_warnings.ModuleWrapper",
  "h5py.h5py_warnings.import_module",
  "h5py.h5r",
  "h5py.h5s",
  "h5py.h5t",
  "h5py.h5z",
  "h5py.highlevel",
  "h5py.ipy_completer",
  "h5py.ipy_completer._retrieve_obj",
  "h5py.ipy_completer.h5py_attr_completer",
  "h5py.ipy_completer.h5py_completer",
  "h5py.ipy_completer.h5py_item_completer",
  "h5py.ipy_completer.load_ipython_extension",
  "h5py.ipy_completer.re_attr_match",
  "h5py.ipy_completer.re_item_match",
  "h5py.ipy_completer.re_object_match",
  "h5py.is_hdf5",
  "h5py.new_enum",
  "h5py.new_vlen",
  "h5py.register_driver",
  "h5py.registered_drivers",
  "h5py.run_tests",
  "h5py.special_dtype",
  "h5py.tests",
  "h5py.tests.MODULES",
  "h5py.tests.common",
  "h5py.tests.common.NUMPY_RELEASE_VERSION",
  "h5py.tests.common.TestCase",
  "h5py.tests.common.UNICODE_FILENAMES",
  "h5py.tests.common.closed_tempfile",
  "h5py.tests.hl",
  "h5py.tests.hl.test_dims_dimensionproxy.TestItems",
  "h5py.tests.mname",
  "h5py.tests.old",
  "h5py.tests.run_tests",
  "h5py.tests.ut",
  "h5py.unregister_driver",
  "h5py.version",
  "h5py.version._H5PY_VERSION_CLS",
  "h5py.version.api_version",
  "h5py.version.api_version_tuple",
  "h5py.version.hdf5_built_version_tuple",
  "h5py.version.hdf5_version",
  "h5py.version.hdf5_version_tuple",
  "h5py.version.info",
  "h5py.version.version",
  "h5py.version.version_tuple"
 ]
}